src/Resources/config/resource-rest.xml

```diff
@@ -7,7 +7,7 @@
     <services>
 
         <service id="cmf_resource_rest.controller.resource" class="Symfony\Cmf\Bundle\ResourceRestBundle\Controller\ResourceController">
-            <argument type="service" id="serializer" />
+            <argument type="service" id="jms_serializer" />
```

Comment:
did jms serializer 1 replace the `serializer` service of symfony, and jms 2 no longer does it? is jms_serializer defined by the jms bundle version 1 as well?

Reply:
Yes, version 1 always defined the `jms_serializer` service, but had a config setting (defaulted to true) that aliased `jms_serializer` to `serializer`. In version 2, this config setting has been removed, so the serializer can only be referenced through the `jms_serializer` service